refs/legacy.json

```diff
@@ -16,7 +16,7 @@
     "IANAPERMHEADERS": "<cite><a href=\"http://www.iana.org/assignments/message-headers/perm-headers.html\">Permanent Message Header Field Names</a></cite>. IANA.",
     "ICC32": "<a href=\"http://www.color.org/icc32.pdf\"><cite>ICC Profile Format Specification, version 3.2.</cite></a> 1995. URL: <a href=\"http://www.color.org/icc32.pdf\">http://www.color.org/icc32.pdf</a>",
     "ICC42": "<a href=\"http://color.org/ICC1v42_2006-05.pdf\"><cite>Image technology colour management &#8212; Architecture, profile format, and data structure.</cite></a> International Color Consortium. Specification ICC.1:2004-10 (Profile version 4.2.0.0) With errata incorporated, 5/22/2006. URL: <a href=\"http://color.org/ICC1v42_2006-05.pdf\">http://color.org/ICC1v42_2006-05.pdf</a> ",
-    "ID-MGM": "Rundle, M. <a href = 'http://identityproject.lse.ac.uk/mary.pdf'><cite>International Data Protection and Digitial Identity Management Tools</cite></a> Internet Governance Forum 2006 - Athens, Privacy Workshop I. 31 October 2006. URL: http://identityproject.lse.ac.uk/mary.pdf ",
+    "ID-MGM": "Rundle, M. <a href = 'https://web.archive.org/web/20191227054302/identityproject.lse.ac.uk/mary.pdf'><cite>International Data Protection and Digital Identity Management Tools</cite></a> Internet Governance Forum 2006 - Athens, Privacy Workshop I. 31 October 2006. URL: https://web.archive.org/web/20191227054302/identityproject.lse.ac.uk/mary.pdf ",
```

Comment:
This repository occasionally uses IA links, and since I had to use IA to validate / correct this content, it seemed like a good idea to pin the URL as well (since the document is no longer available from the original source).